.github/workflows/build-and-test-django.yml

```
@@ -0,0 +1,21 @@
+name: Run Django tests
+
+on:
+  pull_request:
+
+jobs:
+  django-tests:
+    name: Build and test Django production container
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Compose
+        uses: docker/setup-compose-action@v1
+      - name: Copy test environment variables
+        run: cp .env.example .env
+      - name: Build django-prod container
+        run: docker compose build django-prod
+      - name: Run Django tests
```

You may want to check for missing migrations before running the tests (in the spirit of 'fail early'), in case we end up with an elaborate test suite that takes a while to run only to find out that you're missing a migration.

(This is a moot optimisation suggestion; feel free to ignore!)

---

Makes sense!